src/lib/types/Schedule.ts

```diff
@@ -13,6 +13,11 @@ export interface EventSchedule {
 		link: string;
 	};
 	url?: string;
+	hideFromList?: boolean;
+	calendar?: {
+		hasCustomColor: boolean;
+		color: string;
```

Comment:
tbh we should never need the `hasXxx` variables in DB, only make the `color?: ` variable optional here for example.

This comes up a lot